cgo.go

```diff
@@ -1,7 +1,10 @@
 package v8go
 
-// #cgo CXXFLAGS: -fno-rtti -fpic -std=c++14 -I${SRCDIR}/deps/include
+// #cgo CXXFLAGS: -fno-rtti -fpic -std=c++14
+// #cgo !windows CXXFLAGS: -I${SRCDIR}/deps/include
+// #cgo windows CXXFLAGS: -DV8_COMPRESS_POINTERS -DV8_31BIT_SMIS_ON_64BIT_ARCH
```

Comment:
Looks like this is need for *nix too; will update once in master

Reply:
Are you referring to the pointer compression defines? If so, I can change CXXFLAGS to
```
// #cgo CXXFLAGS: -fno-rtti -fpic -std=c++14 -DV8_COMPRESS_POINTERS -DV8_31BIT_SMIS_ON_64BIT_ARCH
// #cgo !windows CXXFLAGS: -I${SRCDIR}/deps/include
```

Let me know if you want me to do this.